docs/cookbook/modifying-a-template-in-administration.md

```diff
@@ -0,0 +1,133 @@
+                    {% endblock %}
+                {% endembed %}
+        
+            {{ form_end(form) }}
+        
+        {% endblock %}
+        ...
+    ```
+    
+    View in the administration before the modification:
+    
+    ![Admin product detail before](img/modifying-a-template-product-before.png)
+
+    Now, add the simple condition into `app/Resources/ShopsysFrameworkBundle/views/Admin/Content/Product/detail.html.twig` to display the product transfer status
+
+    ``` 
+        <div class="form-line">
+            <div class="form-line__line form-line__line--no-space">
+                <div class="form-line__item form-line__item--text">
+                    {% if product.isTransferred  %}
+                        {{ 'Products is already transferred into IS.'|trans }}
+                    {% else %}
+                        {{ 'Products is not yes transferred into IS.'|trans }}
+                    {% endif %}
+                </div>
+            </div>
+        </div>
+    ```
+
+    The template after the modification:
+    
+    ```
+        ...
+        {% block menu_panel_visibilities %}{% endblock %}
+        {% endblock %}
+        
+        {% block breadcrumb %}{% endblock %}
+        
+        {% block main_content %}
+            {% if isMultidomain() %}
+                {% set cssClassOnMultidomainInputs = 'input--domain' %}
+            {% else %}
+                {% set cssClassOnMultidomainInputs = '' %}
+            {% endif %}
+        
+            <div class="form-line">
+                <div class="form-line__line form-line__line--no-space">
+                    <div class="form-line__item form-line__item--text">
+                        {% if product.isTransferred  %}
+                            {{ 'Products is already transferred into IS.'|trans }}
+                        {% else %}
+                            {{ 'Products is not yes transferred into IS.'|trans }}
+                        {% endif %}
+                    </div>
+                </div>
+            </div>
+            
+            {{ form_start(form) }}
+                {{ form_errors(form) }}
+        
+                {% embed '@ShopsysFramework/Admin/Inline/FixedBar/fixedBar.html.twig' %}
+                    {% block fixed_bar_content %}
+                        <button id="js-close-without-saving" type="button" class="btn-link-style">{{ 'Close without saving'|trans }}</button>
+                        {{ form_save(product|default(null), form) }}
+                    {% endblock %}
+                {% endembed %}
+        
+            {{ form_end(form) }}
+        
+        {% endblock %}
+        ...
+    ```
+    
+    View in the administration after the modification:
+    
+    ![Admin product detail after](img/modifying-a-template-product-after.png)
+
+## Conclusion
+On a practical example, we have shown you how to extend the twig template in the administration.
```

Comment:
Maybe something like "You just overwrote the template". Mainly because in most of the document you talk about "you", but also because you give them a feel that they did something great.

Reply:
I understand this approach. But, We have multiple cookbooks and installation guides and any of them contains something like "You just did this...". I think it is because the current state looks more professional and more like the documentation than some blog article. What do you think?

Reply:
ok